examples/xml-namespaces/xml_namespaces.md

```diff
@@ -1,9 +1,9 @@
 # XML namespaces
 
-The goal is to support namespaces without adding complexity if you don’t use them. The 
-`ns:x` qualified name in XML is expanded into `{url}x` where `url` is the namespace name bound to `ns`. The XML namespace
-declarations are kept as attributes using the standard binding of [xmlns](http://www.w3.org/2000/xmlns/).
+The `ns:x` qualified name in XML is expanded into `{url}x` where `url` is the namespace name bound to `ns`. The XML namespace declarations are kept as attributes using the standard binding of [xmlns](http://www.w3.org/2000/xmlns/).
+
+XML namespaces are supported in such a way that they do not add complexity if you do not use them. 
 
 ::: code xml_namespaces.bal :::
 
```

Comment:
Don't we need to say that executing above code, give the below result? I have seen this in new format

Reply:
I believe that's only for the first one or two BBEs. We had it in the original BBEs too, but decided to remove because it was kinda repetitive and not adding value IIRC.